guide/16-geoenrichment-guides/part5_generate_reports.ipynb

```diff
@@ -0,0 +1,928 @@
+{
```

Comment:
<p>The report parameter here means "Optional parameter to identify the id of the report. This may be one of the many default reports available along with our demographic data collections or a customized report.&nbsp;" Please provide explanation and links to the original doc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>